internal/service/batch/job_definition.go

```diff
@@ -69,9 +70,9 @@ func ResourceJobDefinition() *schema.Resource {
 				ValidateFunc: validName,
 			},
 			"node_properties": {
-				Type:     schema.TypeString,
-				Optional: true,
-				ForceNew: true,
+				Type:          schema.TypeString,
+				Optional:      true,
+				ConflictsWith: []string{"eks_properties"},
```

Comment:
```suggestion
				ConflictsWith: []string{"eks_properties", "container_properties"},
```

Reply:
`TestAccBatchJobDefinition_createTypeContainerWithBothProperties` and `TestAccBatchJobDefinition_createTypeMultiNodeWithBothProperties` fails when I add this.

Reply:
Those tests expect failure. Could we just eliminate them?

Reply:
Yeah, let me update the string its looking to match on.